include/grpc/grpc_security.h

```diff
@@ -240,8 +240,11 @@ GRPCAPI grpc_channel_credentials* grpc_ssl_credentials_create(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
     const verify_peer_options* verify_options, void* reserved);
 
+/** Sets the min TLS version in a channel credentials object. */
```

Comment:
Please follow the style of the other comments in this file, and detail the requirements on the arguments.

Similarly for the other comments.

Reply:
I'm not sure what else I could add to this, since the function is very small. I could add `grpc_channel_credentials should not be NULL` but some of the examples below don't do that as well. Do you have any suggestions?
```
/** Creates a credentials object from a plugin with a specified minimum security
 * level. */
GRPCAPI grpc_call_credentials* grpc_metadata_credentials_create_from_plugin(
    grpc_metadata_credentials_plugin plugin,
    grpc_security_level min_security_level, void* reserved);
```
```
/** Creates an IAM credentials object for connecting to Google. */
GRPCAPI grpc_call_credentials* grpc_google_iam_credentials_create(
    const char* authorization_token, const char* authority_selector,
    void* reserved);
```

Reply:
Sure, here's a suggested edit for setting the min TLS version:

/** Sets the minimum TLS version that will be negotiated during the TLS handshake. The caller MUST ensure that |creds| is an instance of |ssl_credentials|.
      - creds: an instance of |ssl_credentials|. If |creds| is nullptr, then this is a no-op.
       - min_tls_version: the minimum TLS version. */

Reply:
Thanks for the suggestion! I've made the changes.